scene/debugger/scene_debugger.cpp

```diff
@@ -141,6 +142,28 @@ void SceneDebugger::_handle_embed_input(const Ref<InputEvent> &p_event, const Di
 	}
 }
 
+Error SceneDebugger::_config_capture(void *p_user, const String &p_msg, const Array &p_args, bool &r_captured) {
+	r_captured = false;
+
+	SceneTree *scene_tree = SceneTree::get_singleton();
+	if (!scene_tree) {
+		return ERR_UNCONFIGURED;
+	}
+	ERR_FAIL_COND_V(p_args.is_empty(), ERR_INVALID_DATA);
+
+	bool valid = true;
+	if (p_msg == "debug_collisions_hint") {
+		scene_tree->set_debug_collisions_hint(p_args[0]);
+		PhysicsServer2D::get_singleton()->emit_signal("_debug_options_changed");
+		// PhysicsServer3D::get_singleton()->emit_signal("_debug_options_changed"); // TODO
```

Comment:
```suggestion
		//PhysicsServer3D::get_singleton()->emit_signal("_debug_options_changed"); // TODO
```
No space for commented out code